src/ucp/core/ucp_request.h

```diff
@@ -79,6 +79,19 @@ enum {
 
 
 /**
+ * Multirail rendezvous-get info.
+ * In regular rndv-get way there is always number of rkeys
+ * is equal to rndv lanes, but in case if tag offloading is
+ * available - only one rkey is proceeded
+ */
+typedef struct ucp_rndv_get_rkey {
+    ucp_lane_index_t  lane_num; /* number of rkeys obtained from peer */
+    ucp_lane_index_t  lane_idx;
```

Comment:
pls add a comment to describe this field

Reply:
added